packages/server/test/e2e/4_before_browser_launch_spec.ts

```diff
@@ -35,4 +35,10 @@ describe('e2e before:browser:launch', () => {
     sanitizeScreenshotDimensions: true,
     snapshot: true,
   })
+
+  e2e.it('can change browser prefs (download dir)', {
+    spec: '*',
```

Comment:
there's no snapshot, and the spec is *, which means there's no assertion that the expected spec actually ran, just that there was some specfile, so i think you should add `snapshot: true` here

Reply:
fair